### PR TITLE
UIF-91 Update create/edit Fund record form

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
Update create/edit Fund record form
https://issues.folio.org/browse/UIF-91
## Screenshot
image
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/components/Fund/FundForm/FundForm.js [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+                >
+                  {metadata && <ViewMetaData metadata={metadata} />}
+                  <Row>
+                    <Col
+                      data-test-col-name
+                      xs={3}
+                    >
+                      <Field
+                        component={TextField}
+                        label={<FormattedMessage id="ui-finance.fund.information.name" />}
```
Comment (aliaksei-chumakou) [inline src/components/Fund/FundForm/FundForm.js]: name isn't a required field?

File: src/components/Fund/FundForm/FundForm.js [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+    const item = find(funds, { id: option }) || option;
+
+    if (!item) return option;
+
+    return item.name;
+  };
+
+  const filter = (filterText, list) => {
+    const renderedItems = filterText
+      ? funds
```
Comment (aliaksei-chumakou) [inline src/components/Fund/FundForm/FundForm.js]: should it be `funds`, or you can filter passed in `list`? I'm not familiar what comes as a `list` though

File: src/components/Fund/FundForm/FundForm.js (hunk 1 above)
Comment (AndreiShumski2) [reply src/components/Fund/FundForm/FundForm.js]: Missed it. thx.

File: src/components/Fund/FundForm/FundForm.js (hunk 2 above)
Comment (AndreiShumski2) [reply src/components/Fund/FundForm/FundForm.js]: Yes. `list` it's an array of fund ids only

**Pull request merged**